lib/components/steps/Home.tsx

```diff
@@ -1,42 +1,36 @@
-import { Box, Button, Typography, Chip } from '@mui/material';
-import { useCdeContext } from "../../CdeContext.tsx";
-import { STEPS } from "../../models.ts";
-import { validateInputMappings, processInputMappings } from '../../services/mappingService.ts';
-import { ModalLayout } from "../layout/ModalLayout.tsx";
-import { StyledTable } from '../common/StyledTable.tsx';
-import { CircleChipDefault, CircleChipSuccess } from '../../icons/index.tsx';
-import { vars } from '../../theme/variables.ts';
-
-const { primary600, gray600 } = vars;
+import {Box, Button, Typography, Chip} from '@mui/material';
+import {useCdeContext} from "../../CdeContext.tsx";
+import {STEPS} from "../../models.ts";
+import {areMappingsValid, getMappingsDict} from '../../services/mappingService.ts';
```

Comment:
@Aiga115 suggestion: if the data is in the context then we don't need this ... we should then instead use the context

Reply:
Question addressed in https://github.com/MetaCell/cde-mapper/pull/12/files/ab7021052fb67f33d62e6695ef249dbc98fddbb9#r1452609886